### PR TITLE
7.x wcag

### PR DESCRIPTION
**Pull request by MorganDawe**
**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-2217)

#WCAG Islandora Solution Pack Entities

aXe accessibility engine: https://www.deque.com/axe/
Web Accessibility inititive: https://www.w3.org/WAI/intro/wcag

# What does this Pull Request do?
The goal of this pull request is to bring islandora solution pack entities into compliance with WCAG 2.0 standards.

It is important to note that this pull aims to address accessibility violations as seen by the average, non authenticated users. Administrative pages remain unaffected.

# What's new?
Adds alt attributes to the thumbnail in the templates and adds corresponding <dd> tag where applicable and a corresponding label.

# How should this be tested?

The easiest way to test these changes locally:
- install the aXe browser plugin in your browser of choice (found here: https://www.deque.com/axe/).
- Recommended: Enable the ‘AT Subtheme’ found in the Adaptive Core (https://www.drupal.org/project/adaptivetheme). This theme is the closest to matching wcag 2.0 requirements i could find.
- Run the aXe tool on a person or department page, or another wcag 2.0 compliance testing toolkit, and examine the page for WCAG 2.0 violations sourced from this modules code.

# Interested parties
Tag (@rosiel @Islandora/7-x-1-x-committers)

### CODE REVIEW

**Review by qadan — APPROVED** [review]

**Comment by MorganDawe** [comment]
@rosiel @Islandora/7-x-1-x-committers Any further suggestions. Reminder this pull is here.

**Comment by MorganDawe** [comment]
@rosiel @Islandora/7-x-1-x-committers do you think you will have time to take another peek at this sometime soon? Not trying to bug. Thank you.

**Comment by rosiel** [comment]
Hey - sorry, still trying to get that release out.

**Comment by MorganDawe** [comment]
@rosiel  Just a reminder that this pull is here. Thank you for taking the time to review it.

File: theme/islandora-person.tpl.php [hunk 1]
```diff
@@ -35,6 +34,10 @@
   </div>
 
   <div class="islandora-object-content">
+    <h2 class="bio"><?php print t('Biography'); ?></h2>
```
Comment (DiegoPino) [inline theme/islandora-person.tpl.php]: This is new? Is the biography related to WCAG? If so, where is this defined?

**Review by DiegoPino — CHANGES_REQUESTED** [review]
Tiny change. I don't see where biography thing is relevant to Wcag and what to do if label is empty == no tn_alt

**Comment by MorganDawe** [comment]
Removed the BIO label.

**Comment by MorganDawe** [comment]
@DiegoPino Just a reminder this pull is here, if you have any further suggestions.

**Review by DiegoPino — APPROVED** [review]

**Comment by DiegoPino** [comment]
OK, cool. Seeing the changes. Approved, merging in 24. Thanks (sorry, was not reading all my notifications, my fault)

File: theme/islandora-dept.tpl.php [hunk 2]
```diff
@@ -9,8 +9,11 @@
 <?php if (isset($variables['tn'])): ?>
   <dl class="islandora-object-tn islandora-department">
     <dt>
-      <img src="<?php print $variables['tn']; ?>"/>
+      <img alt="<?php print $tn_alt;?>" id="<?php print $tn_id;?>" src="<?php print $variables['tn']; ?>"/>
```
Comment (DiegoPino) [inline theme/islandora-dept.tpl.php]: sorry @MorganDawe was about to merge and saw this..:( just for the sake of consistency, could you please change $tn_alt to $variables['tn_alt', etc?
I know they get auto-magically expanded, but makes reading, customizing later much easier if referenced from  $variables array.

**Review by DiegoPino — CHANGES_REQUESTED** [review]
tiny little change.. sorry man

**Comment by MorganDawe** [comment]
@DiegoPino i have updated the ```islandora-dept.tpl.php``` template to reflect the changes you suggested.

**Comment by DiegoPino** [comment]
@MorganDawe ok cool. will wait for Travis to finish, thanks man!

**Review by DiegoPino — APPROVED** [review]

**Comment by DiegoPino** [comment]
All good. Merging in 24 hours! Thanks for addressing all my concerns, i appreciate it

**Comment by MorganDawe** [comment]
@DiegoPino "marco" :)

**Comment by DiegoPino** [comment]
Polo? Merged!

**Pull request merged**